pkgs/development/python-modules/imgtool/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, cbor2
+, click
+, cryptography
+, intelhex
+}:
+
+buildPythonPackage rec {
+  pname = "imgtool";
+  version = "1.10.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.9";
```

Comment:
```suggestion
  disabled = pythonOlder "3.6";
```
https://github.com/mcu-tools/mcuboot/blob/23d28832f02dcdc18687782c6cd8ba99e9b274d2/scripts/setup.py#L15C14-L15C14

Reply:
Done, but it won't build with python 3.8 because the cryptography module doesn't (even though it should), because of backports-zoneinfo which doesn't build. However, this is a cryptography module problem.